website/src/site/site.xml

```diff
@@ -51,7 +51,7 @@ under the License.
     <href></href>
   </bannerLeft>
   <bannerRight>
-    <src>apache-feather.gif</src>
+    <src>apache-feather.png</src>
```

Comment:
I see. I think merging-to-master (except publishing) is enough at this time since we will publish 0.14 soon.